lib/Digitick/Sepa/DomBuilder/BaseDomBuilder.php

```diff
@@ -60,7 +60,6 @@ function __construct($painFormat)
         $this->root = $this->doc->createElement('Document');
         $this->root->setAttribute('xmlns', sprintf("urn:iso:std:iso:20022:tech:xsd:%s", $painFormat));
         $this->root->setAttribute('xmlns:xsi', "http://www.w3.org/2001/XMLSchema-instance");
-        $this->root->setAttribute('xsi:schemaLocation', "urn:iso:std:iso:20022:tech:xsd:$painFormat $painFormat.xsd");
```

Comment:
What about making the schemaLocation optional by a constructor param like __construct(..., $withSchemaLocation = true)